docs/getting-started.md

```diff
@@ -2,32 +2,84 @@
 
 # Getting Started
 
-## Integrating Uno and Warp in your project
+This documentation will guide you through the process of integrating UnoCSS and Warp into your project. UnoCSS is a utility-first CSS framework, and Warp is a design system built on top of UnoCSS.
+
+## 1. Integrate with UnoCSS and Warp
 
 ### Install
 
+To get started, you need to install the necessary packages. At the moment, you should install the `alpha` versions of the `@warp-ds/uno` packages until the major versions are available.
+
+#### Using npm:
+
+```shell
+npm install -D @warp-ds/uno@alpha unocss
+```
+
+#### Using pnpm:
+
 ```shell
-pnpm add -D @warp-ds/uno unocss
+pnpm add -D @warp-ds/uno@alpha unocss
 ```
 
-> Webpack based projects should also add: `@unocss/webpack`
+> If you are using a Webpack-based project, you also need to add `@unocss/webpack` (see [UnoCSS docs](https://unocss.dev/integrations/webpack) for more information)
 
 ### Configure
 
 Create an `uno.config.js` file with the following content. This file will configure Uno, but we still need to get Uno into your project.
 
 ```js
-import { defineConfig } from 'unocss'
-import { presetWarp } from '@warp-ds/uno'
-
+import { defineConfig } from "unocss";
+import { presetWarp } from "@warp-ds/uno";
 export default defineConfig({
-  presets: [
-    presetWarp()
-  ]
-})
+  presets: [presetWarp()],
+});
 ```
 
 #### Adding Uno to your build tool
 
-For now, please refer to the [examples](https://github.com/unocss/unocss/tree/main/examples) found in the UnoCSS project. We will eventually have in-depth install guides for frameworks on the golden path.
+By default, UnoCSS will automatically look for uno.config.{js,ts,mjs,mts} or unocss.config.{js,ts,mjs,mts} in the root directory of your project. If you prefer to specify the config file manually, you can do so. Here's an example for Vite:
+
+```ts
+// vite.config.js
+import { defineConfig } from "vite";
+import UnoCSS from "unocss/vite";
+import { presetWarp } from "@warp-ds/uno";
+import { classes } from "@warp-ds/component-classes/classes";
```

Comment:
This is something relevant to those who are using Warp components, so can be removed here.

Reply:
Wups, nice catch!